include/ScratchViews.h

```diff
@@ -86,6 +112,8 @@ class ScratchViews
 
   std::vector<ViewHolder*> fieldViews;
 
+  std::vector<MasterElementViews> meViews;
+  std::vector<bool> hasCoordField;
```

Comment:
This is a private field and does not require STL container behavior. So I am fine with `vector<bool>` here.